ChatApp.xcodeproj/project.pbxproj

```diff
@@ -127,6 +155,54 @@
 			path = ChatAppUITests;
 			sourceTree = "<group>";
 		};
+		8FF84BFF29F94C3000A815E8 /* View */ = {
+			isa = PBXGroup;
+			children = (
+				8F80CCC529F0586100F121BD /* ViewController.swift */,
+				8FF84C3529FBFD9300A815E8 /* ViewControllerConstants.swift */,
+			);
+			path = View;
+			sourceTree = "<group>";
+		};
+		8FF84C0029F94C3D00A815E8 /* Model */ = {
+			isa = PBXGroup;
+			children = (
+				8F80CCF129F17BDC00F121BD /* ChatViewModel.swift */,
+			);
+			path = Model;
+			sourceTree = "<group>";
+		};
+		8FF84C0129F94C5F00A815E8 /* Extension+UI */ = {
```

Comment:
When having folder of extensions for native UIKit classes - just "Extensions" is OK